src/common/Gateway.sol

```diff
@@ -127,7 +127,7 @@ contract Gateway is Auth, Recoverable, IGateway {
 
         emit PrepareMessage(centrifugeId, poolId, message);
 
-        uint128 gasLimit = gasService.gasLimit(centrifugeId, message) + extraGasLimit;
+        uint128 gasLimit = gasService.messageGasLimit(centrifugeId, message) + extraGasLimit;
```

Comment:
Why did we rename this?

Reply:
I'd rather minimize changes and stick to current name

Reply:
If you still have a super strong opinion, I can revert this, but I would say the change is minimal (?)

Reply:
I get it, seems okay then